src/restconfig/src/test/java/org/geoserver/rest/security/XStreamParserTest.java

```diff
@@ -0,0 +1,37 @@
+/* (c) 2015 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.rest.security;
+
+import static org.junit.Assert.assertNotNull;
+
+import com.thoughtworks.xstream.XStream;
+import com.thoughtworks.xstream.io.json.JettisonMappedXmlDriver;
+import org.geoserver.rest.security.xml.JaxbUser;
+import org.junit.Test;
+
+public class XStreamParserTest {
+
+    // https://osgeo-org.atlassian.net/browse/GEOS-8486
+    @Test
+    public void jsonReadWriteTest() {
```

Comment:
Please add an integration test too, for both the XML and JSON case, the reason why we are dealing with this regression is the lack of such test.